### DISCUSSION TITLE
Cannot run AppImage in Alpine Linux

### DISCUSSION THREAD

**Discussion by nitin065** [ImageMagick/ImageMagick#6726] [Contribute]
I am trying to run appimage version in alpine linux, but I am getting `not found` issue.

I have described about the issue in details [Here](https://unix.stackexchange.com/questions/616576/cannot-run-magick-appimage-binary-in-linux)

**Comment by dlemstra** [comment]
The AppImage is build with libc instead of musl. And because Alpine uses musl you cannot run the executable there.

**Comment by nitin065** [comment]
@dlemstra Can I manually build it using musl? If it is a simple approach?

**Comment by dlemstra** [comment]
Maybe we could move the current creation of the AppImage to a container build? Then we could also do another container build that uses a musl image.

**Comment by nitin065** [comment]
@dlemstra Any update on when this topic could be picked up? This AppImage would help us a lot :)

**Comment by dlemstra** [comment]
It can be picked up by anyone that wants to work on this. We cannot give an ETA when or if that will happen.
